external-api/package.json

```diff
@@ -19,22 +19,14 @@
   "main": "src/index.ts",
   "scripts": {
     "api-test": "jest --coverage --verbose --forceExit --runInBand --config=api-test.jest.config.js",
-    "lint": "eslint ./src ./api-tests --ext .ts",
+    "lint": "eslint ./src ./api-tests --ext .ts && npm run type-check",
```

Comment:
Same as before

Reply:
As discussed, will raise a separate PR to review lint scripts to make them more general.